### DISCUSSION TITLE
Guess Why There Will Never Be Seasonal MMR Reset

### DISCUSSION THREAD

**Discussion by cedricchi** [ValveSoftware/Dota2-Gameplay#25071] [Polls]
Why not solve all the boosters and smurf problems with a single easy MMR reset?
We could motivate all players to play Dota again, and all the pro player don't have to smurf any more.
Cast your vote and perhaps read the following.

As player counts hit 5-years-low (https://steamdb.info/app/570/charts/), GG team posted this matchmaking meme https://www.youtube.com/watch?v=y79u72uhI4c
![image](https://github.com/user-attachments/assets/f36de81d-1ca2-4029-aa61-43885851a71f)

We all know every other game has Seasonal MMR Reset, except Dota 2 which has obviously rigged 50% (if you have Dota+) win rate that you can't break however good you are. The "dark toxic system" manipulates your games in all ranks, forcing you to lose just when you have 3 games win streak by giving you ridiculous bots.
Somehow, Valve decided one win + one loss = mmr +1. Therefore, if you buy an account you will lose very very very slowly, perhaps even gain some mmr in the long run because skill has absolutely nothing to do with MMR. In other word, Dota 2 is boosters and account buyers' warm bed. Besides, ranking up in solo queue is impossible, we can only seek help from boosters who are doing just great in Dota 2.

Some facts and evidence:
bots playing fake games in Immortal,

Is this a normal Immortal match?
https://www.reddit.com/r/DotA2/comments/1ilqaw0/is_this_a_normal_immortal_match/
byu/Zerudinha inDotA2

boosters have made it to Immortal 4 rank.

Rank 04 Wintrading Along Map Hacker
https://www.reddit.com/r/DotA2/comments/1ikkp8k/rank_04_wintrading_along_map_hacker/
byu/Meepomon inDotA2
https://www.dotabuff.com/matches/8181832861

For more, just look at your games and see how ridiculous they are

More Immortal account than the lower ranks combined, boosters are using bots in SEA and EU they have reached 20,000 MMR
![image](https://github.com/user-attachments/assets/89baea91-91ab-4700-a67a-6a7d2a229845)

**Comment by ogresleftheadisgay** [comment]
No idiot, the only reason there is no seasonal MMR is cuz the top players would not quit crying about their 10k+ disappearing. Valve is scared that their loyal players will leave. And I dont think theres necessarily a balck magic mechanic to pull you down if you win too much, I think there is dogshit matchmaking and no deterrents for players to not play like absolute selfish assholes or throw willingly. those 2 things are hindering dota from growing.

**Reply by cedricchi** [reply]
So you think by keeping MMR player count will be stable? Do you see the player count right now? How long it takes you to find a match? When you get a match, is the quality good or bad?
If there is no "black magic" why in your word it is dogshit? Why the BS system is not doing its work in term of stopping people trolling or jungle farming like a bot when playing pos 5? What if BS system does work? Will you be able to find a single player to play with?
If matchmaking and toxic behaviour are hindering Dota, why Valve do nothing?

**Comment by ogresleftheadisgay** [comment]
I want mmr to be seadonal, im telling you the reason why it isnt. Learn to read
